SpatialGDK/Source/SpatialGDKTests/SpatialGDK/LoadBalancing/LayeredLBStrategy/LayeredLBStrategyTest.cpp

```diff
@@ -204,11 +205,17 @@ bool FCleanup::Update()
 		Pair.Value->Destroy(/*bNetForce*/ true);
 	}
 
+	GEditor->RequestEndPlayMap();
+
 	return true;
 }
 
 LAYEREDLBSTRATEGY_TEST(GIVEN_strat_is_not_ready_WHEN_local_virtual_worker_id_is_set_THEN_is_ready)
 {
+	const auto Settings = GetMutableDefault<USpatialGDKEditorSettings>();
```

Comment:
Why is this required in these tests but not `OwnershipLockingPolicyTest`?

Reply:
Mainly because `OwnershipLockingPolicyTest` was not recognised as flaky, but I can have a look to see if it requires similar changes 😄

Reply:
Seconding Danny, would be good to look at the other tests which utilize AutomationOpenMap in the same pass.